### PR TITLE
feat: rabbitmq

### PR DESCRIPTION
**Pull request by ElderMatt**
Otomi-Api: [501](https://github.com/redkubes/otomi-api/pull/501)
Otomi-Core: [1498](https://github.com/redkubes/otomi-core/pull/1498)
Otomi-Console: [382](https://github.com/redkubes/otomi-console/pull/382)
Redkubes.io: [54](https://github.com/redkubes/redkubes.github.io/pull/54)

If you would like to test the feature on a cluster please do the following:
- Deploy Otomi with the `matthew-feat-rabbitmq` branch
- Wait till it is done
- Go to Gitea
- Push rabbitMQ workload to gitea repo
    - In the `otomi-charts` repo checkout `matthew/feat/rabbitmq`
- In the vs code terminal do the following
    - git remote add cluster GITEA_CHARTS_REPO
        - adds that repo as current git branch
￼
Pasted Graphic 1

    - export GIT_SSL_NO_VERIFY=true; 
        - allows to be able to push to branch if it is unprotected
    - git push -f cluster
        - push current branch to gitea
    - git push -f cluster matthew/feat/rabbitmq:main
        - push `matthew/feat/rabbitmq` branch to `main` Gitea branch  with `force` tag

### CODE REVIEW

File: otomi-quickstart-rabbitmq/values.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+# if you want to check all spec options, please check the rabbitmqCluster crd
+# https://github.com/rabbitmq/cluster-operator/blob/main/config/crd/bases/rabbitmq.com_rabbitmqclusters.yaml
+rabbitmqCluster:
+  spec:
+    # affinity:
+    #   nodeAffinity: {}
+    #   podAffinity: {}
+    #   podAntiAffinity: {}
+    delayStartSeconds: 30
+  #   override: {}
```
Comment (srodenhuis) [inline otomi-quickstart-rabbitmq/values.yaml]: The required indenting is unclear. Please use one space after the #

File: otomi-quickstart-rabbitmq/values.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
+# Default values for otomi-quickstart-rabbitmq.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+imagePullSecrets: []
+nameOverride: ""
```
Comment (srodenhuis) [inline otomi-quickstart-rabbitmq/values.yaml:5]: Is the nameOverride required? We only use the fullnameOverride to automatically add the name of the workload

Comment (ElderMatt) [reply otomi-quickstart-rabbitmq/values.yaml:5]: It does get used in the _helper.tpl, all the other charts also have nameOverride.

**Review by j-zimnowoda — APPROVED** [review]
Looks good. If you can connect app to RabbitMQ cluster then go ahead and merge it.

**Pull request merged**